Versions.props

```diff
@@ -0,0 +1,19 @@
+<Project>
```

Comment:
I know this is how it's usually done in the .NET repos, but I'd rather just inline this into Directory.Packages.props personally

Reply:
I believe that won't work. Try it.